packages/calypso-e2e/src/lib/pages/account-closed-page.ts

```diff
@@ -0,0 +1,30 @@
+import { Page } from 'playwright';
+
+const selectors = {
+	accountClosedMessage: ':text("Your account has been closed")',
+	accountClosedSubmessage: ':text("Thanks for flying with WordPress.com")',
```

Comment:
Optional: do you think we need this submessage? This feels like something quirky we might tweak semi-regularly, and I wonder if it's enough to just confirm the main closure message.

Reply:
Hmm, you're right. Let's rely on less selectors.